module2/persistence/src/test/java/com/epam/esm/dao/InMemoryDbConfig.java

```diff
@@ -0,0 +1,79 @@
+package com.epam.esm.dao;
+
+import com.epam.esm.config.JdbcConfig;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.web.SpringJUnitWebConfig;
+import org.springframework.test.context.support.AnnotationConfigContextLoader;
+
+import java.sql.SQLException;
+
+@SpringJUnitWebConfig(JdbcConfig.class)
+@ContextConfiguration(
+        classes = {JdbcConfig.class},
+        loader = AnnotationConfigContextLoader.class)
+public class InMemoryDbConfig {
+    private static final String CREATE_TAG_TABLE = "CREATE TABLE IF NOT EXISTS tag (" +
```

Comment:
Why do we have this create table statements in code while we have init.sql in the connection url?